sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -1720,4 +1720,11 @@ object functions {
      UnresolvedFunction(udfName, cols.map(_.expr))
   }
 
+  /**
+   * Parsers the comma separated expression strings into the columns they represent, similar to
+   * DataFrame.selectExpr
```

Comment:
can you add groups for this?

Also I think this should just return a single Column. Otherwise users have to keep doing : _*